src/test/java/com/axlabs/neo/grantshares/GrantSharesTreasuryTest.java

```diff
@@ -100,4 +98,57 @@ public void getFunders() throws IOException {
                 denise.getECKeyPair().getPublicKey().getEncoded(true)));
     }
 
-}
+    @Test
+    public void removedFunders() throws Throwable {
```

Comment:
In this project the tests have relatively expressive names which mach the proposal description used in the test. Thus, I propose changing the name to `execute_proposal_with_remove_founder`.